docs/src/test-webserver-js.md

```diff
@@ -37,6 +37,7 @@ export default defineConfig({
 | `stdout` | If `"pipe"`, it will pipe the stdout of the command to the process stdout. If `"ignore"`, it will ignore the stdout of the command. Default to `"ignore"`. |
 | `stderr` | Whether to pipe the stderr of the command to the process stderr or ignore it. Defaults to `"pipe"`. |
 | `timeout` | How long to wait for the process to start up and be available in milliseconds. Defaults to 60000. |
+| `kill` | How to shut down the process gracefully. If unspecified, the process group is forcefully `SIGKILL`ed. If set to `{ SIGINT: 500 }`, the process group is sent a `SIGINT` signal, followed by `SIGKILL` if it doesn't exit within 500ms. You can also use `SIGTERM` instead. A `0` timeout means no `SIGKILL` will be sent. Windows doesn't support `SIGINT` and `SIGTERM` signals, so this option is ignored. |
```

Comment:
I'd split it into 2 fields`{ signal: SIGINT', timeout 5000 }` to make it clear that only one signal can be specified and what the value semantics is.

Reply:
Should we also rename it from `kill` to `shutdown` or `gracefulShutdown`? At the moment, `kill: undefined` means "KILL immediately", which could be surprising to some.

Reply:
refactored in https://github.com/microsoft/playwright/pull/34130/commits/1361b4b495dfe5ba946542cf317e24240abf34c6